### ISSUE TITLE
textAngular in form - tab to go to next <input> tag

### ISSUE THREAD

**Issue by SamuelMarks** [fraywing/textAngular#1150]
How do I set this up?

**Comment by pbassut** [comment]
Wait, do you want to navigate through inputs inside textAngular or between textAngular instances?

**Comment by SamuelMarks** [comment]
So given a simple form like this:
```
<form ng-submit="namespace.create()">
  <div text-angular ng-model="namespace.newWhatever.rich"
          data-placeholder="rich" data-name="rich"
          class="ta-text ta-editor form-content"
          data-ta-toolbar="[['bold', 'italics', 'underline', 'strikeThrough', 'ul', 'ol']]">
    </div>
    </div>
    <label for="foo">Foo</label>
          <input type="text" id="foo" ng-model="namespace.newWhatever.foo">
</form>
```

I want to tab from the textAngular input to the text input (and back).

**Comment by pbassut** [comment]
Hmm. I'm afraid that's currently not supported by textAngular.
Unless you listen for a keydown event and jump to the next input.

**Comment by SamuelMarks** [comment]
Hmm... well maybe if we got this directive working on a [`textarea`](https://www.w3.org/wiki/HTML/Elements/textarea) then all these use-cases would be viable

**Comment by pbassut** [comment]
Did you try to use it on a textarea? It should work fine.

**Comment by SamuelMarks** [comment]
Yeah, didn't work at all with `textarea`

**Comment by pbassut** [comment]
I don't think that a RTE should intercept the tab key when in fact it's supposed to tab(add spaces).
As I said, this is not currently supported by textAngular and I think you could make your way through if you intercept a keydown/up event.
